src/temp/Project/Filter.tsx

```diff
@@ -0,0 +1,107 @@
+import FilterIcon from '../icons/FilterIcon';
+import { Button } from '@mui/material';
+import style from './Filter.module.scss';
+import { useTranslations } from 'next-intl';
+import { useState } from 'react';
+
+export type Classification =
+  | 'large-scale-planting'
+  | 'agroforestry'
+  | 'natural-regeneration'
+  | 'managed-regeneration'
+  | 'urban-planting'
+  | 'other-planting';
+
+interface FilterProps {
+  filterApplied: Classification | undefined;
+  setFilterApplied: (newValue: Classification | undefined) => void;
+  availableFilters: Classification[];
+}
+
+export const FilterDropDown = ({
+  filterApplied,
+  setFilterApplied,
+  availableFilters,
+}: FilterProps) => {
+  const tAllProjects = useTranslations('AllProjects');
+
+  const handleFilterSelection = (
+    filterItem: Classification | undefined
+  ): void => {
+    setFilterApplied(filterItem);
+  };
+
+  return (
+    <>
+      {availableFilters.length > 0 ? (
+        <div className={style.projectListMainContainer}>
+          <button
+            className={style.filterButton}
+            onClick={() => handleFilterSelection(undefined)}
+          >
+            <div
+              className={
+                filterApplied === undefined
+                  ? style.filterSelected
+                  : style.projectName
+              }
+            >
+              {tAllProjects('classificationTypes.all')}
+            </div>
+            <hr className={style.hrLine} />
+          </button>
+          <div className={style.container}>
```

Comment:
`container` is too generic, perhaps rename to `filterOptionsSubContainer`. Makes it easier to identify where these styles are coming from.